torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -235,6 +239,10 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                     full_node = g.call_function(
                         ttnn.full, args=(arg_metadata.size(),), kwargs=new_kwargs
                     )
+                    # Intermediate node meta is not preserved, this ensures retention
+                    # Are output dims same for full node and actual node to be replaced?
```

Comment:
I think output dims are not guaranteed to be the same due to tiling

Reply:
I think the meta field stores non-tiled output shape for each of these.